### PR TITLE
add some functionality

### PR DESCRIPTION
**Pull request by lsimmons1832**

### CODE REVIEW

**Comment by lsimmons1832** [comment]
@devoisaurus

File: JS/DOMHandler.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+var finalPrice = document.getElementById("finalPrice");
+
+// Define button
+var button = document.getElementById("makeMySandwich");
+
+// Get a reference to the <select> element that has all the meat options
+var meatChooser = document.getElementById("meat");
+var breadChooser = document.getElementById("bread");
+var cheeseChooser = document.getElementById("cheese");
+var condimentChooser = document.getElementById("condiment");
```
Comment (devoisaurus) [inline JS/DOMHandler.js]: Check your element name in the HTML and make sure it matches what you're referencing here

File: JS/DOMHandler.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+});
+
+breadChooser.addEventListener("click", function(event) 
+  // Get the value chosen from the DOM
+  selectedTopping = event.target.value;
+    sandichMaker.addBread(selectedTopping);
+	
+	if (event.target.checked === false){
+		selectedTopping = event.target.value;
+		sandichMaker.removeMeat(selectedTopping);
```
Comment (devoisaurus) [inline JS/DOMHandler.js]: Should this be removeMeat, or reference the component of the function?

File: JS/condiments.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+   	yellowmustard: 0.50,
+   	spicymustard: 0.15,
+   	mayo: 0.20,
+   	relish: 0.50,
+   	honeymustard: 0.30,
+   	ranch: 0.35
+   };
+
+// Augment the original object with another method
+   maker.addVeggies = function() {
```
Comment (devoisaurus) [inline JS/condiments.js]: what is this function adding?

**Review by devoisaurus — COMMENTED** [review]
I've left some notes for particular lines to look at

File: JS/DOMHandler.js (hunk 1 above)
Comment (lsimmons1832) [reply JS/DOMHandler.js]: The id's specified match the index file

File: JS/condiments.js (hunk 3 above)
Comment (lsimmons1832) [reply JS/condiments.js]: Nice catch.  I forgot to rename the function after copying

**Comment by lsimmons1832** [comment]
@devoisaurus

**Comment by devoisaurus** [comment]
Looks great!   I like the button styling, too!  👍

**Pull request merged**